requirements/doc.txt

```diff
@@ -5,5 +5,5 @@ tqdm>=4.11
 nb2plots>=0.5
 sphinxcontrib-bibtex
 sphinx<2
-sphinx_rtd_theme
+pydata-sphinx-theme>=0.8.1
```

Comment:
Where do the list of doc requirements live? Build failing due to missing dependency.
(from workflow: `python -m pip install .[all,testing,docs]` )
@jni

Reply:
They are in setup.cfg